lib/index.js

```diff
@@ -31,8 +31,20 @@ export default {
     this.subscriptions.add(atom.config.observe('flow-ide.onlyIfAppropriate', (onlyIfAppropriate) => {
       this.onlyIfAppropriate = onlyIfAppropriate
     }))
+
+    let priority = atom.config.get('flow-ide.hyperclickPriority')
```

Comment:
The current version seems to provide the right behavior for me:
* It doesn't prompt when Atom starts
* It only captures the value on the first pass
* It only prompts when the value is different and there isn't already a notification.